src/twisted/python/__init__.py

```diff
@@ -17,6 +17,13 @@
     "Please use constantly from PyPI instead.",
     "twisted.python", "constants")
 
+
+deprecatedModuleAttribute(
+    Version("Twisted", "NEXT", 0, 0),
```

Comment:
> Just a side comment: is this NEXT thing documented somewhere?

Yup! [Right here](https://github.com/hawkowl/incremental#updating)  It may be possible to change this (though I don't, for the record), but that change would be in incremental.